components/UI/ServiceTable.js

```diff
@@ -42,13 +89,17 @@ const TableCell = styled.td`
 
 `;
 
-export function ServiceTable({ children }) {
+export function ServiceTable({ children, selected, align }) {
```

Comment:
When are these being used? Can't find any consumers. Shouldn't `ServiceTable` be just this?
```js
const ServiceTable = Table;
```